pkgs/games/unnethack/default.nix

```
@@ -22,6 +24,13 @@ stdenv.mkDerivation rec {
                    ];
 
   makeFlags = [ "GAMEPERM=744" ];
+  patches = [
+    (fetchpatch {
```

usually with patches, there's a comment on why it's necessary, and when it can be removed
```suggestion
  patches = [
    # fix regression with bison, merged in master
    (fetchpatch {
```

---

Then there should be a comment in the manual on that. But there isn't: https://github.com/NixOS/nixpkgs/blob/master/doc/contributing/coding-conventions.xml#L862

---

In the case of the manual, the name of the patch gives away the intent. although we should probably add that maintainer-ship is a lot easier when the patches mention whether they should be removed when bumping.